Src/BootCamp.Chapter/CleanTable.cs

```diff
@@ -0,0 +1,202 @@
+
+                if (i < skipCarriageReturn)
+                {
+                    buildLines.Append("\r\n");
+                }
+            }
+
+            File.WriteAllText(_fixedBalanceFile, buildLines.ToString());
+        }
+
+        private static bool IsValidLine(string line)
+        {
+            var words = line.Split(",");
+
+            if (!IsValidName(words[0]))
+            {
+                throw new InvalidBalancesException($"Invalid Name {words[0]}");
+            }
+
+            for (var i = 1; i < words.Length; i++)
+            {
+                if (!IsValidBalance(words[i]))
+                {
+                    throw new InvalidBalancesException($"Invalid Balance {words[i]}");
+                }
+            }
+
+            return true;
+        }
+
+        private static bool IsValidName(string name)
+        {
+            for (var i = 0; i < name.Length; i++)
+            {
+                if (!IsValidLetter(name[i]) && !HasPeriodAtEnd(name))
+                {
+                    return false;
+                }
+            }
+
+            return true;
+        }
+
+        private static bool IsValidLetter(char letter)
+        {
+            if (IsValidAlphabet(letter) || IsValidSymbolForName(letter))
+            {
+                return true;
+            }
+            return false;
+        }
+
+        private static bool IsValidAlphabet(char letter)
+        {
+            if ((letter >= 'a' && letter <= 'z') || (letter >= 'A' && letter <= 'Z'))
+            {
+                return true;
+            }
+            return false;
+        }
+
+        private static bool IsValidSymbolForName(char letter)
+        {
+            if ((letter == ' ') || (letter == '\'') || (letter == '-'))
+            {
+                return true;
+            }
+            return false;
+        }
+
+        private static bool HasPeriodAtEnd(string name)
+        {
+            if (name[name.Length - 1] == '.')
+            {
+                return true;
+            }
+            return false;
+        }
+
+        private static bool IsValidBalance(string balance)
+        {
+            for (var i = 0; i < balance.Length; i++)
+            {
+                if (balance[i] != DollarSign && !IsValidNumber(balance[i]) && !IsValidSymbolForBalance(balance[i]))
+                {
+                    return false;
+                }
+            }
+
+            return true;
+        }
+
+        private static bool IsValidNumber(char number)
+        {
+            if (number >= '0' && number <= '9')
+            {
+                return true;
+            }
+            return false;
+        }
+
+        private static bool IsValidSymbolForBalance(char punctuation)
+        {
+            if (punctuation == '-' || punctuation == '.')
+            {
+                return true;
+            }
+            return false;
+        }
+
+        private void DisplayStasticalData(string[] peopleBalances)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'DisplayStasticalData' a static method.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5063788)